src/probnum/diffeq/odefiltsmooth/ivpfiltsmooth.py

```diff
@@ -140,11 +144,11 @@ def _estimate_local_error(self, pred_rv, t_new, calibrated_diffmat, **kwargs):
             Statistics and Computing, 2019.
         """
         local_pred_rv = Normal(pred_rv.mean, calibrated_diffmat)
-        local_meas_rv, _ = self.gfilt.measure(t_new, local_pred_rv, **kwargs)
+        local_meas_rv, _ = self.gfilt.measure(t_new, local_pred_rv)
         error = local_meas_rv.cov.diagonal()
-        return error
+        return np.sqrt(error)
```

Comment:
it needs to be the square-root of the diagonal, no? Or am I missing something? Please tell me if I am wrong.

Reply:
I guess we might want a unittest for adaptive steps that checks that the ratio maxstep/minstep (which is called separation radius, btw) is within known bounds (e.g. roughly equal to 100 for a set algorithm choice). If this square thing would happen, then the step-sizes would change enough.

Reply:
I will open an issue